checks.d/zk.py

```diff
@@ -211,3 +225,25 @@ def parse_stat(self, buf):
         metrics.append(('zookeeper.nodes', long(value.strip())))
 
         return metrics, tags, mode
+
+
+    def parse_mntr(self, buf):
+        ''' `buf` is a readable file-like object
+            returns a tuple: (metric_hash, state)
+        '''
+        metric_hash = {}
+        state = ''
+
+        buf.seek(0)
+        buf.readline() # skip version line
+
+        for line in buf:
+            data = line.split()
+            if len(data) == 2 and data[0] == 'zk_server_state': # state is not an int
+                state = data[1].lower()
+            elif len(data) == 2:
```

Comment:
@Sirupsen Should I handle errors like the ones above? We don't want the entire thing to not report if just the `mntr` command fails?